lib/inspec/cli.rb

```diff
@@ -194,6 +194,8 @@ def detect
     desc: 'A single command string to run instead of launching the shell'
   option :format, type: :string, default: nil, hide: true,
     desc: 'Which formatter to use: cli, documentation, html, json, json-min, junit, progress'
+  option :depends, type: :array, default: [],
```

Comment:
What do you think about calling the option `--load-profiles` instead of `--depends`? I know `--depends` is technically correct re: implementation and consistent with the inspec.yml, but I wonder if `--load-profiles` is a clearer option as to what it feels like to the user. Don't feel obligated to take this suggestion, just wanted to toss it out there.

Reply:
Load-profiles is interesting too yeah 🤔 

To add a small argument: "load profiles" would technically only load the libraries of the profile and not the controls, i.e. not the entire profile. I was pondering "load libraries" (but that may lead users to specify the libraries folder which we don't want either; traversing dependencies for all resources is beautiful right now)

There may be a better term...

Reply:
`--load-profile-libs`?

Reply:
I can't think of anything terribly better. If `--load-profile-libs` doesn't feel right, and you think `--depends` is clear enough, please merge when ready. :)  We can always add an alias if someone comes up with something better.

Reply:
Good idea, i'll make sure to add the docs you suggested below. Otherwise we can deprecate and find something else. Naming is hard...